dir.props

```diff
@@ -17,7 +17,7 @@
   <!-- Trying to mitigate the number of warnings that the solution contains by
     adding this into our Release builds.  As a result, it should be caught in
     our PRs because of the CI builds -->
-  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+  <!-- <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
Another option is to suppress the CS1701 warning.... but I am afraid that it will result in suppressing any binding redirect errors that may happen.